Traceback (most recent call last):
  File "/home/giovanni/.config/sublime-text/Installed Packages/LSP.sublime-package/plugin/documents.py", line 902, in clear_async
  File "/home/giovanni/.config/sublime-text/Installed Packages/LSP.sublime-package/plugin/session_view.py", line 71, in on_before_remove
  File "/home/giovanni/.config/sublime-text/Installed Packages/LSP.sublime-package/plugin/code_lens.py", line 98, in clear_view
  File "/opt/sublime_text/Lib/python33/sublime.py", line 1892, in update
    for phantom, region in zip(self.phantoms, regions):
TypeError: zip argument #2 must support iteration
Traceback (most recent call last):
  File "/home/giovanni/.config/sublime-text/Installed Packages/LSP.sublime-package/plugin/documents.py", line 902, in clear_async
  File "/home/giovanni/.config/sublime-text/Installed Packages/LSP.sublime-package/plugin/session_view.py", line 71, in on_before_remove
  File "/home/giovanni/.config/sublime-text/Installed Packages/LSP.sublime-package/plugin/code_lens.py", line 98, in clear_view
  File "/opt/sublime_text/Lib/python33/sublime.py", line 1892, in update
    for phantom, region in zip(self.phantoms, regions):
TypeError: zip argument #2 must support iteration
